Makefile

```diff
@@ -98,7 +98,7 @@ go-apidiff:
 ##@ Tests
 
 .PHONY: test
-test: test-unit test-integration test-testdata test-book ## Run the unit and integration tests (used in the CI)
+test: test-unit test-integration test-testdata test-book test-license ## Run the unit and integration tests (used in the CI)
```

Comment:
Could we have a GA /ci job new only for this one to make it easier for us to know when/if fails?
I mean, i new target + GitHub actions?

Reply:
So do you want a Github actions in addition to the script? I thought we wanted one or the other, otherwise it would be repetitive.

Reply:
The suggestion would be to call this one only in a GA that does this check.  
Not call the test twice at all. That would mean not adding `test-license` in this target also. 

The reason would be to make it easier for contributors to check why/what is falling in the ci. 
However, that is just nit. It is passing on the tests now so I am ok with too.